src/main/webapp/WEB-INF/html/productsList.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+
+    <meta charset="UTF-8">
+    <title>Shop Application</title>
+    <script src="https://code.jquery.com/jquery-3.5.0.js"></script>
+
+
+</head>
+<body>
+    <div>
+        <div>
+            <h2>Shop</h2>
+        </div>
+    </div>
+    <div>
+        <div>
+            <table id="products">
+                <tr>
+                    <th>Product Name</th>
+                    <th>Price PLN</th>
+                    <th>Action</th>
+                </tr>
+
+            </table>
+        </div>
+    </div>
+    <div>
+        <div>
+            <button id="addToCart">Cart</button>
+        </div>
+    </div>
+    <form id="idForm" method="post">
+        <input type="hidden" id="idInput" name="productIds">
```

Comment:
Sending this data using this form was a walk-around. You no longer use the hidden so it can be removed.